internal/install/recipe_installer.go

```diff
@@ -266,6 +268,19 @@ func (i *RecipeInstaller) installRecipes(ctx context.Context, m *types.Discovery
 			log.Debugf("Failed while executing and validating with progress for recipe name %s, detail:%s", r.Name, err)
 			log.Warn(err)
 			log.Warn(i.failMessage(r.DisplayName))
+
+			if _, ok := err.(*types.UnsupportedOperatingSytemError); ok {
```

Comment:
@sanderblue  I dont think we need this logic block, since we already have a return with this logic on line 262?

Reply:
Will double check. Thanks

Reply:
Hmm, I think you're right. Testing this it out now.